Library/Formula/pinentry-mac.rb

```diff
@@ -1,10 +1,9 @@
-require "formula"
-
 class PinentryMac < Formula
   homepage "https://github.com/GPGTools/pinentry-mac"
   url "https://github.com/GPGTools/pinentry-mac/archive/v0.8.1.tar.gz"
   sha256 "79aaa11fa8076ff335b3a1f41c230ef7c8435a757705e6484199f562f26b490f"
   head "https://github.com/GPGTools/pinentry-mac.git"
+  revision 1
```

Comment:
Meh, let's bin the revision. It doesn't seem worth forcing a reinstall for as whatever code could have been run will have been already.